blackbox/stratisd.spec

```diff
@@ -61,7 +78,7 @@ mv %{buildroot}%{_bindir}/stratis_uuids_to_names %{buildroot}%{udevdir}/stratis_
 
 %if %{with check}
 %check
-%cargo_test -- --skip real_ --skip loop_ --skip travis_
+%cargo_test -- -- --skip real_ --skip loop_ --skip travis_
```

Comment:
If I don't add that, it will fail the "cargo test" command on the Fedora test machine with this error:
```
+ /usr/bin/env CARGO_HOME=.cargo RUSTC_BOOTSTRAP=1 /usr/bin/cargo test -j2 -Z avoid-dev-deps --release --no-fail-fast --skip real_ --skip loop_ --skip travis_
error: Found argument '--skip' which wasn't expected, or isn't valid in this context

USAGE:
    cargo test --jobs <N> --no-fail-fast --release -Z <FLAG>...

For more information try --help
error: Bad exit status from /var/tmp/rpm-tmp.jVfQLl (%check)
```

It's an unfortunate difference between the two `%cargo` macros, rust-packaging in Fedora, and rust-toolset in RHEL.  But if I add `-- --` to that line, both pass successfully.

Reply:
Can you put in a comment here, then?

(To me, it looks kind of like there's a bug in the Fedora macro. And that could be a problem, because there could be further changes that make those two macros entirely incompatible. I'll try to come up with a way to pursue that question.)

Reply:
(Coming here from the rust2rpm issue.) This is not a bug. The first `--` separates macro arguments from `cargo test` arguments, and the second `--` separates `cargo test` arguments from test runner arguments (of which `--skip` is one).